.circleci/config.yml

```diff
@@ -189,15 +189,14 @@ jobs:
       - setup
       - run: yarn cache clean
       - run: yarn install
-      - run:
-          name: Run react unit tests
-          command: yarn test --maxWorkers=2 --reporters=default --reporters=jest-junit
-          environment:
-            JEST_JUNIT_OUTPUT_DIR: ./reports/junit/
-      - store_test_results:
-          path: ./reports/junit/
```

Comment:
note: this config was written to address some issues in CircleCI. I've reverted it to be in line with the LAP repo's config, and it seems to be working fine.